Cargo.toml

```diff
@@ -36,6 +36,7 @@ gdk4wayland = { package = "gdk4-wayland", version = "0.9", optional = true }
 gdk4x11 = { package = "gdk4-x11", version = "0.9", optional = true }
```

Comment:
before i forget, please move the "relates to #38" part of the commit header to the commit body